roles/basic-site/tasks/main.yml

```diff
@@ -21,4 +21,51 @@
   with_items:
     - www
     - log
-    - cert
\ No newline at end of file
+    - cert
+
+- name: generate list of DNS entries
+  set_fact:
+    dns: "{{ redirects }} + ['{{ server_name }}']"
+
+- name: cert/dns entries
+  copy:
+    dest: /var/www/{{ site }}/cert/dns
+    content: |
+      {% for entry in dns %}
+      {{ entry }}
+      {% endfor %}
+    backup: true
+  notify:
+    - generate cert
+
+- name: nginx config
+  when: setup_html_vhost is defined
```

Comment:
Nothing seems to actually set this? If it's not used, then I would remove it.

Reply:
I left it in intentionally as a "basic template". But you are right, it is not used anywhere and could be removed...